src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,6 +5,16 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Don't begin class or method implementation with empty lines. Remove the empty lines before the class definition.